fix(docs): improve `kestra.plugins` clarity (#2428)
kestra-io/docs: content/docs/configuration/index.md

          password: ${kestra.ee.license.fingerprint:}
```

### Plugin Defaults

> You can provide global plugin defaults using the `kestra.plugins.defaults` configuration. Those provide default values for plugins properties and will be applied to each task on your cluster **if a property is not defined** on flows or tasks. Plugin defaults allow ensuring a property is defined at a default value for these tasks.

Reviewer: ```suggestion
You can provide global plugin defaults using the `kestra.plugins.defaults` configuration. These provide default values for plugin properties and will be applied to each task on your cluster **if a property is not defined** on flows or tasks. Plugin defaults ensure a property is defined as a default value for these tasks.
```

```yaml
kestra:
  plugins:
    defaults:
    - type: io.kestra.plugin.core.log.Log
      values:
        level: ERROR
```

For greater granularity, you can use the flow-level [`pluginDefaults`](../04.workflow-components/09.plugin-defaults.md#plugin-defaults-on-a-flow-level) property, which overrides global defaults for the given flow.

::alert{type="info"}
For more information, you can see the dedicated [Plugin ​Defaults](../04.workflow-components/09.plugin-defaults.md) section.
::

#### Forced Plugin Defaults

The `forced` flag ensures a property is set globally for a task, and no task can override it.

You can use this to isolate tasks in containers, such as scripting tasks. For Bash tasks and other script tasks in the core, we advise you to force `io.kestra.plugin.scripts.runner.docker.Docker` isolation and to provide global cluster configuration:

```yaml
kestra:
  plugins:
    defaults:
      - type: io.kestra.plugin.scripts.shell.Commands
        forced: true
        values:
          containerImage: ubuntu:latest
          taskRunner:
            type: io.kestra.plugin.scripts.runner.docker.Docker
```

### Enable or Disable Features

> The `kestra.plugins.configuration` configuration section can be used to enable or disable some features of specific Kestra plugins, or to set some default values for them.

Reviewer: ```suggestion
The `kestra.plugins.configuration` section can be used to enable or disable specific Kestra plugin features or to set default values for them.
```

In this example, the `recoverMissedSchedules` is set to `NONE`, which means that Kestra will not recover any missed schedules for this specific flow regardless of the global configuration.

### Volume Enabled for Docker Task Runner

> Volumes mount are disabled by default for security reasons, you can enabled it with this configurations:

Reviewer: ```suggestion
Volume mounts are disabled by default for security reasons; you can enable them with these configurations:
```

      path: /home/kestra/tmp
```

> Note that the `tmpDir` path must be aligned to the volume path otherwise Kestra will not know what directory to mount for the `tmp` directory.

Reviewer: ```suggestion
Note that the `tmpDir` path must be aligned with the volume path; otherwise, Kestra will not know what directory to mount for the `tmp` directory.
```
